phase1/master.cfg

```diff
@@ -1037,6 +1037,9 @@ def prepareFactory(target):
             descriptionDone="dl/ pruned",
             command="find dl/ -mindepth 1 -atime +15 -delete -print",
```

Comment:
missed the -print -delete reorder :)

Reply:
I've noticed that remark, so I've tried it locally, didn't spot any difference with the output, so simply skipped that.